Microsoft.Toolkit.Win32/Microsoft.Toolkit.Wpf.UI.Controls/Microsoft.Toolkit.Wpf.UI.Controls.csproj

```diff
@@ -0,0 +1,96 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net462</TargetFramework>
+    <RootNamespace>Microsoft.Toolkit.Wpf.UI.Controls</RootNamespace>
+    <AssemblyName>Microsoft.Toolkit.Wpf.UI.Controls</AssemblyName>
+    <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
+    <RunCodeAnalysis>true</RunCodeAnalysis>
+
+    <Title>Windows Community Toolkit WPF Controls</Title>
+    <Description>This library provides XAML controls for WPF. It is part of the Windows Community Toolkit.</Description>
+    <PackageTags>WebView Win32 WPF UWP XAML</PackageTags>
+
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
+    <DefineConstants>TRACE;DEBUG;CODE_ANALYSIS;NET47;DEV_DEBUG;DEBUG_LAYOUT</DefineConstants>
```

Comment:
Update to be `<DefineConstants>$(DefoneConstants);DEV_DEBUG;DEBUG_LAYOUT</DefineConstants>` as with WinForms .csproj